### PR TITLE
V.0.13.0

### PR DESCRIPTION
**Pull request by ageddesi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:12-alpine
```
Comment (benhalverson) [inline Dockerfile:1]: Node.js 12.x is EOL'ed  I'd change this to 16.x
https://nodejs.dev/en/download/

File: app.ts [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 const morgan = require('morgan');
 const cors = require('cors');
 
 const app = express();
 const port = 3000;
 
 // Load Mock Modules
-require('./modules/animal/api/animal-routes')(app) // Animals
-require('./modules/chat/api/chat-routes')(app) // Chat
+require('./modules/animal/api/animal-routes')(app); // Animals
```
Comment (benhalverson) [inline app.ts:12]: why are you using require in a typescript file? Why not `import {thing} from 'thing'`

File: Dockerfile (hunk 1 above)
Comment (ageddesi) [reply Dockerfile:1]: Raised as a ticket #129

File: app.ts (hunk 2 above)
Comment (ageddesi) [reply app.ts:12]: There has been no real conversation around this. This was just the way when I was learning node. If there is a better way, please can you raise a ticket with information in it so we can look into doing that. Thanks @benhalverson 👍

**Pull request merged**